core/src/main/scala/org/apache/spark/network/BlockTransferService.scala

```diff
@@ -83,7 +83,12 @@ abstract class BlockTransferService extends ShuffleClient with Closeable with Lo
    *
    * It is also only available after [[init]] is invoked.
    */
-  def fetchBlockSync(host: String, port: Int, execId: String, blockId: String): ManagedBuffer = {
+  def fetchBlockSync(
+      host: String,
+      port: Int,
+      execId: String,
+      blockId: String,
+      timeout: Int): ManagedBuffer = {
```

Comment:
I have a question about the time unit of such configurations. Do you think it's better to use same time unit for all configurations? Currently, it's hard to know the time unit when reading a configuration file. I usually need to search them in the codes or docs.